test/lambda_toolshed/test_utils.cljc

```diff
@@ -13,10 +13,22 @@
        (reify
          cljs.test/IAsyncTest
          cljs.core/IFn
-         (~'-invoke [_# done2#] (obj# done2#) c#)))
+         (~'-invoke [_# done2#] (obj# done2#))))
     ;; In Clojure we block awaiting the completion of the async test block
     `(async/<!! (async/go (do ~@body)))))
 
+(defmacro test-async
```

Comment:
with the changes this PR makes, is the `go-test` macro needed/used anymore?

Reply:
Not currently.  But it might still prove useful (@stevenproctor recently asked me to use it to demonstrate compatibility with the old approach of returning a channel).  It would be damn hard to re-create... I recall having a difficult time writing it originally.  But, I suppose that is what git is for.

Reply:
that makes sense, having those tests would likely help show more instead of just tell.

yeah, from what I remember of convos at the time, it was pretty gnarly.

kinda like pepperidge farm, git remembers 😃

Reply:
Not really.  I've deleted it.